Without this change leveldb autodetects tcmalloc from sysroot and
sometimes became dependant on gperftools

Disable autodetection without TCMALLOC_ENABLED=true pased to make

--- a/build_detect_platform	2013-07-21 19:56:57.447212944 +0200
+++ b/build_detect_platform	2013-07-21 19:57:54.866214329 +0200
@@ -184,15 +184,17 @@
         PLATFORM_LIBS="$PLATFORM_LIBS -lsnappy"
     fi
 
-    # Test whether tcmalloc is available
-    $CXX $CXXFLAGS -x c++ - -o $CXXOUTPUT -ltcmalloc 2>/dev/null  <<EOF
-      int main() {}
+    if [ "$TCMALLOC_ENABLED" = "true" ]; then
+        # Test whether tcmalloc is available
+        $CXX $CXXFLAGS -x c++ - -o $CXXOUTPUT -ltcmalloc 2>/dev/null  <<EOF
+          int main() {}
 EOF
-    if [ "$?" = 0 ]; then
-        PLATFORM_LIBS="$PLATFORM_LIBS -ltcmalloc"
-    fi
+        if [ "$?" = 0 ]; then
+            PLATFORM_LIBS="$PLATFORM_LIBS -ltcmalloc"
+        fi
 
-    rm -f $CXXOUTPUT 2>/dev/null
+        rm -f $CXXOUTPUT 2>/dev/null
+    fi
 fi
 
 PLATFORM_CCFLAGS="$PLATFORM_CCFLAGS $COMMON_FLAGS"
